.github/workflows/build.yml

```diff
@@ -175,7 +181,9 @@ jobs:
       env:
         GH_TOKEN: ${{ github.token }}
         GH_REPO: ${{ github.repository }}
-      run: gh release edit "${{ needs.release.outputs.tag }}" --draft=false
+      run: |
+        body="jruby/jruby@${{ needs.build.outputs.commit }}"
```

Comment:
We can't just do this because the `prepare` job relies on the commit mentioned here to avoid extra rebuilds

Reply:
I think instead in the prepare job you have to get the latest snapshot commit somehow, then it should work correctly.

Reply:
@eregon From what I see that condition is simply not used, which is why I didn't bother

https://github.com/ruby/jruby-dev-builder/blob/c5cf8fafc93d8b0596fa411bdf7f22d2ecd82e4e/.github/workflows/build.yml#L48-L49

Reply:
Did a complete refactoring of the most part. Added the actual download url to the release note, and use that as the uniqueness check for new builds.